### PR TITLE
update ribowaltz with an extra option

### PR DESCRIPTION
**Pull request by pavanvidem**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by pavanvidem** [comment]
Adding an extra param. No new tests are needed.

**Comment by bgruening** [comment]
Can you please fix the lining?

**Comment by bgruening** [comment]
Still one r lint error.

File: tools/ribowaltz/ribowaltz.R [hunk 1]
```diff
@@ -21,6 +21,7 @@ spec <- matrix(c(
   "cds_coverage_info", "Z", 1, "character",
   "psite_info_rdata", "O", 0, "character",
   "refseq_sep", "s", 0, "character",
+  "indel_threshold", "t", 0, "integer",
```
Comment (bernt-matthias) [inline tools/ribowaltz/ribowaltz.R:24]: the 3rd column is the default or? Should be 5 then...

**Review by bernt-matthias — COMMENTED** [review]
> Adding an extra param. No new tests are needed.

Why? :)

File: tools/ribowaltz/ribowaltz.R (hunk 1 above)
Comment (pavanvidem) [reply tools/ribowaltz/ribowaltz.R:24]: I think it means is optional or required option.

Comment (bernt-matthias) [reply tools/ribowaltz/ribowaltz.R:24]: Right, but according to https://cran.r-project.org/web/packages/getopt/getopt.pdf

```
Column 3: Argument mask of the flag. An integer. Possible values: 0=no argu-
ment, 1=required argument, 2=optional argument.
```

this should be 1 or 2. Maybe you can fix this across the matrix?

Also, this highlights the need for tests! We should make sure that the argument has an effect if set to a non-default value (and we should have done this for all parameters).

I also tend to become lazy and skip tests and docs .. and have to remind myself that code that is not tested usually does not work. Also makes the life of reviewers easier .. and only the life of the CI harder.

Comment (bernt-matthias) [reply tools/ribowaltz/ribowaltz.R:24]: > I think it means ...

I also often tend to make assumptions based on my intuition .. and most of the time I'm wrong :)

Comment (bernt-matthias) [reply tools/ribowaltz/ribowaltz.R:24]: Maybe print the content of `opt` in the R code (maybe to stderr) and check if the parameters are set correctly by using an `assert_stderr`.

Comment (pavanvidem) [reply tools/ribowaltz/ribowaltz.R:24]: You're correct. I initially copied it from DESeq2 wrapper which I thought was well tested. So now we have to fix this in many other R scripts too.

Comment (bernt-matthias) [reply tools/ribowaltz/ribowaltz.R:24]: Actually, I'm wrong https://github.com/trevorld/r-getopt/issues/3 .. LOL 

So all good :)

**Review by bgruening — APPROVED** [review]
I guess a test can not hurt, but looks good to me.

**Review by bgruening — APPROVED** [review]

**Pull request merged**